mapproxy/config/config.py

```diff
@@ -194,7 +196,7 @@ def load_config(config, config_file=None, config_dict=None, clear_existing=False
     defaults = _to_options_map(config_dict)
 
     if defaults:
-        for key, value in iteritems(defaults):
+        for key, value in defaults.items(): # todo
```

Comment:
why todo?

Reply:
I was not sure if the code was correct before. But i will leave it as it was before.

Reply:
```suggestion
        for key, value in defaults.items():
```